### ISSUE TITLE
command not found if argument has a '+' sign like 'md5+bcyrpt'

### ISSUE THREAD

**Issue by a-peyrard** [restx/restx#199]
When I ran: `restx hash md5+bcrypt foo` the shell responded with a `command not found: hash md5`

**Comment by fcamblor** [comment]
Hi Augustin,

Not sure what you're expecting as it sounds more like a "pipe" than anything here.
I assume you would expect following behaviour :
- execute `restx hash md5 foo`
- then execute `restx hash bcrypt` on the previous command execution result (output)

So it is rather a "pipe" than a "and" (+ sign is rather used for "chaining" / "and" commands), meaning that I would use another syntax than the + one for this, something like `restx hash md5 foo | bcrypt`

WDYT ?

**Comment by fcamblor** [comment]
OK, forget about my comment, just saw that there is an "expected" `md5+bcrypt` parameter [here](https://github.com/restx/restx/blob/master/restx-core-shell/src/main/java/restx/core/shell/HashShellCommand.java#L88)

**Comment by fcamblor** [comment]
I think the `md5+bcrypt` (and `sha1+bcrypt`) are colliding with [`+` operator in shell](https://github.com/restx/restx/blob/master/restx-shell/src/main/java/restx/shell/RestxShell.java#L415)

I proposed PR #272 which replaces `md5+bcrypt` by `md5#bcrypt` (and `sha1#bcrypt`)

**Comment by fcamblor** [comment]
Closed PR #272 as we have old users with non-changing habits

tldr; you can still call `sha1+bcrypt`, but you need to call it from within restx shell (and not in your standard shell as it will collice with restx + operator)
